tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -245,112 +302,263 @@ describe('Lobby - P1 Perspective', () => {
-    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-back-card"]').should('exist');
-    cy.window()
-      .its('cuttle.gameStore')
-      .then((store) => {
-        // Test: store state
-        expect(store.p1Ready).to.eq(true); // Player is ready
-        expect(store.opponentIsReady).to.eq(false); // Opponent is not ready
-        // Click Unready button
-        cy.get('[data-cy=ready-button]')
-          .should('contain', 'UNREADY')
-          .click()
-          .should('not.contain', 'UNREADY')
-          .should('contain', 'READY');
-          cy.get('[data-cy=my-indicator]').find('[data-cy="lobby-back-card"]').should('exist');
-        //Return updated store state
-        return cy.wrap(store);
-      })
-      .then((updatedGameState) => {
-        //Test updated store state
-        expect(updatedGameState.p1Ready).to.eq(false); // Player not ready
-      });
+  it('join, get ready, reload', function () {
+    playerGetsReady();
+    cy.reload();
+    cy.url().should('include', '/lobby');
+      indicatorShowsNameAndCard('player');
+      userShouldDisplayReady('player', true);
+      indicatorShowsNameAndCard('opponent');
+      userShouldDisplayReady('opponent', false);
+      testReadyValueInState('p1', true);
+      testReadyValueInState('p0', false);
+      checkOpponentIsReadyInGetter(false);
+  });
+
+  it('join, opponent gets ready, reload', function () {
+    cy.readyOpponent();
+    cy.reload();
+    cy.url().should('include', '/lobby');
+      indicatorShowsNameAndCard('player');
+      userShouldDisplayReady('player', false);
+      indicatorShowsNameAndCard('opponent');
+      userShouldDisplayReady('opponent', true);
+      testReadyValueInState('p1', false);
+      testReadyValueInState('p0', true);
+      checkOpponentIsReadyInGetter(true);
   });
 
   it('Game starts when both players are ready - opponent ready before joining', function () {
-    cy.get('[data-cy=exit-button]').click(); // leave game so opponent can ready before player joins
+    playerLeavesLobby(); // leave game so opponent can ready before player joins
     cy.readyOpponent();
-    // Join game again
+    //player joins back
     cy.window()
       .its('cuttle.gameStore')
       .then((store) => store.requestSubscribe(this.gameSummary.gameId));
-    cy.vueRoute(`/lobby/${this.gameSummary.gameId}`);
-    cy.get('[data-cy=ready-button]').click();
-    // Test that game started
-    assertGameStarted();
+      cy.vueRoute(`/lobby/${this.gameSummary.gameId}`);
+      playerGetsReady();
+      assertGameStarted();
   });
 
-  it('Game starts when both players are ready - opponent readies first after player joins', () => {
+  it('Game starts when both players are ready - opponent readies first after player joins', function () {
     cy.readyOpponent();
-    cy.get('[data-cy=ready-button]').click();
-    assertGameStarted();
+    playerGetsReady();
+      assertGameStarted();
```

Comment:
whitespace
```suggestion
    assertGameStarted();
```

Reply:
when writing the file, I indented with 1 white space to create visual separation between the actions and the tests.   Click button, make some verification.  I can see how that might not be a standard practice, but it felt much easier on the eyes and more readable. 
I can take the white space out, no problem.  Would there be a better way to make it easier to read while following good practices?